lib/components/viewers/stop-viewer.js

```diff
@@ -71,7 +71,7 @@ class StopViewer extends Component {
 
```

Comment:
When clicking on a related stop, clicking back dumps you back to the trip planner instead of the previous stop.